lib/screens/authentication/recover/recover_account_found/interactor/usecases/fetch_recover_guardian_initial_data.dart

```diff
@@ -21,7 +22,8 @@ class FetchRecoverGuardianInitialDataUseCase {
       recoveryPrivateKey = settingsStorage.privateKey!;
     } else {
       recoveryPrivateKey = EOSPrivateKey.fromRandom().toString();
-      settingsStorage.enableRecoveryMode(accountName: accountName, privateKey: recoveryPrivateKey);
```

Comment:
we could make a use case for enable recovery mode - but since it was the exact same as save account plus that flag - I don't think it adds to the code?!

LMK what you think

Reply:
I like your solution with the use cases, I would also include the flag (privateKeyBackedUp) in them.
They give more clarity than creating functions in the settings storage

Reply:
Ok done